apps/aecore/src/aec_sync.erl

```diff
@@ -1,7 +1,7 @@
 %%%=============================================================================
 %%% @copyright 2017, Aeternity Anstalt
 %%% @doc
-%%%    Module storing peers list and providing funtionc for peers interaction
+%%%    Module storing peers list and providing funtions for peer interaction
```

Comment:
functions

Reply:
I like doing improvements in small steps, let's leave some low hanging fruit for the next person editing :-)